sunpy/visualization/animator/tests/test_mapsequence.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+import pytest
+
+from astropy.visualization.wcsaxes import WCSAxes
+
+import sunpy.data.test
+import sunpy.map
+from sunpy.tests.helpers import figure_test
+from sunpy.visualization.animator import MapSequenceAnimator
+
+testpath = sunpy.data.test.rootdir
+
+
+@pytest.fixture
+def test_map_sequence():
+    return sunpy.map.Map(
+        os.path.join(
+            testpath, 'aia_171_level1.fits'), os.path.join(
+            testpath, 'mdi_fd_Ic_6h_01d.5871.0000_s.fits'), sequence=True)
+
+
+def test_construct_map_sequence_animator(test_map_sequence):
+    map_animator = MapSequenceAnimator(test_map_sequence)
+    assert isinstance(map_animator, MapSequenceAnimator)
+
+
+@figure_test
+def test_map_sequence_animator_wcs_simple_plot(test_map_sequence):
+    map_animator = MapSequenceAnimator(test_map_sequence)
+    return map_animator.fig
+
+
+def test_axes(test_map_sequence):
+    map_animator = MapSequenceAnimator(test_map_sequence)
+    assert isinstance(map_animator.axes, WCSAxes)
+
+
+@figure_test
+def test_map_sequence_animator_wcs_update_plot(test_map_sequence):
+    map_animator = MapSequenceAnimator(test_map_sequence)
+    map_animator.updatefig(1, map_animator.im, map_animator.sliders[0]._slider)
+    return map_animator.fig
+
+
+@figure_test
+def test_map_sequence_animator_wcs_colorbar_buttons(test_map_sequence):
+    bf = [lambda x: x] * 10
+    bl = ['button'] * 10
+    map_animator = MapSequenceAnimator(
+        test_map_sequence,
+        colorbar=True,
+        button_func=bf,
+        button_labels=bl)
+    map_animator.updatefig(
+        1,
+        map_animator.im,
+        map_animator.sliders[0]._slider)
+    return map_animator.fig
+
+
+@figure_test
+def test_map_sequence_animator_wcs_colorbar_buttons_default_labels(
```

Comment:
What are these tests testing?

Reply:
This just tests whether the display of the buttons are happening, drawing parrallels from arrayanimator like I previously mentioned.

Reply:
I see, do we need 2 of them?

Reply:
Also, not too sure if getting the buttons from the matplotlib figure are possible (for testing via code isntead of figure tests).

Reply:
I guess my point of view is that, we should map out what the API is of MapsequenceAnimator and work towards testing as much of it a we can. Ideally with code if we can and using figure test where either a pure code test is difficult/impossible or we want to verify something purely visual.

Reply:
Hmm okay, the arrayanimator class is also tested this way with respect to the buttons and figures, and I couldn't find any other method of testing the buttons through code. Do you have any suggestions?

Reply:
No, I've not worked on these kind of tests before nor am I too familiar with MapsequenceAnimator.

Reply:
Okay then, I shall let this be. I'll add 2 more methods which test a few more functions of MapsequenceAnimator in a bit and then you can make a decision on what to do.

Reply:
I feel like these two tests that deal with the "wcs_colorbar" probably don't really add anything.

Reply:
Should I get rid of them then?

Reply:
Yeah, I don't see the point in them. Especially having two of them.

Reply:
The same base animator is tested in array animator so I guess I shall remove them.